packages/minecraft.rb

```diff
@@ -3,50 +3,31 @@
 class Minecraft < Package
   description 'A sandbox video game developed by Mojang Studios AB.'
   homepage 'https://minecraft.net'
-  version '2.2.1867'
+  version '2.2.3200'
   license 'Mojang'
   compatibility 'x86_64'
   source_url 'https://launcher.mojang.com/download/Minecraft.deb'
-  source_sha256 '9a8b6b75db199461ae241fbee6adf65fc6636204c58eb79feff968e307464727'
+  # The minecraft-launcher deb changes often, and we are downloading via https
+  # so maybe it is ok to trust the source package without further hashing.
+  source_sha256 `curl -Ls #{source_url} | sha256sum | tail -n1 | cut -d' ' -f1`.chomp
```

Comment:
This is problematic since it executes with any command.  Not sure this is the best approach.  Would `source_url 'SKIP'` work better?

Reply:
Ah! i see your point...

Reply:
Another way to fix this is to generate the pre-built binary.

Reply:
I doubt proprietary software like minecraft allows for a license to redistribute....

Reply:
I think we can handle this with `'SKIP'` but we just need to modify crew to have 
```
Digest::SHA256.hexdigest( File.read(cachefile) ) == sha256sum || sha256sum =~ /^SKIP$/i
```
and
```
Digest::SHA256.hexdigest( File.read(filename) ) == sha256sum || sha256sum =~ /^SKIP$/i
```
put in....